index.html

```diff
@@ -1,14 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+ 
+
 </head>
+
 <body>
-    <h1 class="tittle"></h1>
-    
+  <header class="tool_bar">
+
+    <a class="Logo" href="">
+      <h2>My Logo</h2>
+    </a>
+
+
+    <nav>
+      <a href=""> <img class="union" src="images/Union.png" alt=""></a>
+    </nav>
+
+  </header>
+
+  <section class="headline-sec">
```

Comment:
- [ ] Please use flexbox to align the headline section, since flexbox is better for one-directional aligning and it helps when you try to make your code responsive when you try to move to the desktop version look how it is look in larger sizes now.
![image](https://user-images.githubusercontent.com/74744787/219856302-092c50bd-3197-4825-92a8-57d4a02e91a2.png)